src/main/java/com/github/underscore/U.java

```diff
@@ -621,6 +621,34 @@ public boolean contains(final T elem) {
         return contains(list.subList(fromIndex, list.size()), elem);
     }
 
+    public boolean containsAtLeast(final Integer count, final T value) {
+        return U.containsAtLeast(this.iterable, count, value);
+    }
+
+    public boolean containsAtMost(final Integer count, final T value) {
+        return U.containsAtMost(this.iterable, count, value);
+    }
+
+    public static <E> boolean containsAtLeast(final Iterable<E> values, final Integer count, final E value) {
+        List<E> filtered = filter(values, new Predicate<E>() {
+            @Override
+            public boolean test(E input) {
+                return input == value;
```

Comment:
It should be `equals` check. `return elem == null ? e == null : elem.equals(e);`